src/animations/LiftoffAnimation.tsx

```diff
@@ -44,18 +44,22 @@ const LiftoffAnimation: React.FC<JSX.IntrinsicElements['group']> = ({
       }
     }
   });
-
+  const isMobile = window.innerWidth < 768;
```

Comment:
This won't update if the screen size changes. You may want to look at https://chakra-ui.com/docs/hooks/use-breakpoint-value or https://chakra-ui.com/docs/hooks/use-media-query.